modules/validations/crossing_ways.js

```diff
@@ -701,6 +744,42 @@ export function validationCrossingWays(context) {
         });
     }
 
+    /**
+     * @param {[number, number]} loc
+     * @param {[string, string][]} edges
+     * @param {Tags} connectionTags
+     */
+    function actionConnectCrossingWays(loc, edges, connectionTags) {
```

Comment:
this function is not new, if you click [hide whitespace changes](https://github.com/openstreetmap/iD/pull/10524/files?w=1), then the github UI understands that this function was simply deindented & moved 20 lines up.